applications/RomApplication/tests/test_randomized_singular_value_decomposition.py

```diff
@@ -0,0 +1,30 @@
+#import python class test
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+#import python packages
+from KratosMultiphysics.RomApplication.randomized_singular_value_decomposition import RandomizedSingularValueDecomposition
+import numpy as np
+import numpy.matlib
+
+def synthetic_matrix(degree, rows = 100,repetitions=20):
```

Comment:
just ensure that the test runs very fast, even on a very slow machine

Reply:
Ran it on my potato computer, and the test for the randomized svd takes about 0.05 seconds. 

(the already-in-place tests which launch a ROM simulation and compare its solution with the expected one, do take about 4 seconds in total on this very limited computer)